src/application/modules/cb_query.py

```diff
@@ -1,9 +1,12 @@
 from cb_utilities import *
+import urllib
+import re
+import hashlib
 
 def _get_query_metrics(user, passwrd, node_list, cluster_name=""):
     '''Gets the metrics for the query nodes'''
-    query_info = {}
```

Comment:
Variable definition should be at this scope, not within the _get_completed_query_metrics